aidermacs-backend-comint.el

```diff
@@ -260,7 +260,9 @@ _OUTPUT is the text to be processed."
                (re-search-backward "^\\([^`[:space:]]+\\)$" (line-beginning-position -3) t)))
      (let ((file (match-string 1)))
        (cdr (cl-assoc-if (lambda (re) (string-match re file)) auto-mode-alist))))
-   'fundamental-mode))
+   (progn
+     (message "aidermacs warning: can't detect major mode at %d" (point))
+     'fundamental-mode)))
```

Comment:
Maybe, maybe we can fall-back to markdown-mode?

Reply:
But that will only be markdown mode inside the code block. It won’t see the code fences, so it won’t highlight it as the appropriate language.